ADOPTERS.md

```diff
@@ -2,4 +2,7 @@
 
 A non-exhaustive unordered list of organizations that have adopted OpenTelemetry.
 
-**_[Google Cloud Platform](https://cloud.google.com)_** uses the OpenTelemetry collector as an agent and forwarder. Cloud Trace and Monitoring use the OpenTelemetry SDKs to capture metrics and traces from customers' applications.
+[Google Cloud Platform](https://cloud.google.com) uses the OpenTelemetry collector as an agent and forwarder. Cloud Trace and Monitoring use the OpenTelemetry SDKs to capture metrics and traces from customers' applications.
```

Comment:
I removed decoration to simplify the doc (text already stands out as hyperlink).